### PR TITLE
Add docs for feature flags in Keeper

### PR DESCRIPTION
**Pull request by antonio2368**

### CODE REVIEW

File: docs/en/guides/sre/keeper/index.md [hunk 1]
```diff
@@ -341,6 +341,40 @@ last_snapshot_idx   50
...
+```
+
+### Feature flags
+
+Keeper is fully compatible with ZooKeeper and its clients but it also introduces some unique features and request types that can be used by ClickHouse client. 
+Because those features can introduce backward incompatible change, most of them are disabled by default and can be enabled using `keeper_server.feature_flags` config.  
+All features can be disabled explicitly.  
+If you want to enable a new feature for your Keeper cluster, we recommend you to first update all the Keeper instances in the cluster and then enable the feature.
+
+Example of feature flag config that disabled `multi_read` and enables `check_not_exists`:
```
Comment (DanRoscigno) [inline docs/en/guides/sre/keeper/index.md:359]:
```suggestion
Example of feature flag config that disables `multi_read` and enables `check_not_exists`:
```

File: docs/en/guides/sre/keeper/index.md [hunk 2]
```diff
@@ -341,6 +341,40 @@ last_snapshot_idx   50
...
+    <keeper_server>
+        <feature_flags>
+            <multi_read>0</multi_read>
+            <check_not_exists>1</check_not_exists>
+        </feature_flags>
+    </keeper_server>
+</clickhouse>
+```
+
+Following features are available:
```
Comment (DanRoscigno) [inline docs/en/guides/sre/keeper/index.md:372]:
```suggestion
The following features are available:
```

File: docs/en/guides/sre/keeper/index.md [hunk 3]
```diff
@@ -341,6 +341,40 @@ last_snapshot_idx   50
...
+
+```
+filtered_list   1
+multi_read  1
+check_not_exists    0
+```
+
+### Feature flags
+
+Keeper is fully compatible with ZooKeeper and its clients but it also introduces some unique features and request types that can be used by ClickHouse client.
```
Comment (DanRoscigno) [inline docs/en/guides/sre/keeper/index.md:354]:
```suggestion
Keeper is fully compatible with ZooKeeper and its clients, but it also introduces some unique features and request types that can be used by ClickHouse client. 
```

**Review by DanRoscigno — APPROVED** [review]

**Comment by robot-ch-test-poll3** [comment]
This is an automated comment for commit 3f7c8fb3d8eb1e0b1a56799601039d4cc0aeb547 with description of existing statuses. It's updated for the latest CI running
The full report is available [here](https://s3.amazonaws.com/clickhouse-test-reports/1299/3f7c8fb3d8eb1e0b1a56799601039d4cc0aeb547/ci_running.html)
The overall status of the commit is 🟢 success

<table><thead><tr><th>Check name</th><th>Description</th><th>Status</th></tr></thead>
<tbody><tr><td>Docs Check</td><td>Builds and tests the documentation</td><td>🟢 success</td></tr>
<tr><td>Vercel</td><td>There's no description for the check yet, please add it to tests/ci/ci_config.py:CHECK_DESCRIPTIONS</td><td>🟢 success</td></tr>
</table>

**Pull request merged**